doc/efa-env-var.md

```diff
@@ -0,0 +1,105 @@
+# EFA Cheatsheet
+
+## 1. Settings via environment variables
+
+For optimized performance, you may need to set additional environment variables depending on the
+versions of your libfabric.
+
+<table>
+   <thead>
+      <th>Setting</th>
+      <th>Description</th>
+   </thead>
+   <tr>
+      <td><code>FI_EFA_USE_HUGE_PAGE=0</code></td>
+      <td>Set to 0 when you see <code>os.fork()</code> causes <code>OSError: Cannot allocate
+      memory</code>. Typically happen by multi-process PyTorch data loader. Disabling huge page
+      causes minor performance hit, but it's needed to prevent fork fails due to the operating
+      system running out of huge pages.</td>
+   </tr>
+   <tr>
+      <td><code>FI_EFA_FORK_SAFE=1</code></td>
```

Comment:
I would remove this flag, or at least note that the plugin sets this flag automatically (all reasonably modern versions of the plugin) for versions of Libfabric that support it.

Reply:
I do agree that it's not needed anymore with modern versions, but I'd still leave it here with a note to educate users who set this based on older examples (and there're still many out there).

Reply:
@rauteric : can I clarify if modern versions of this plugin set this env-var automatically for all kernel version? Or, is this env-var still needed for older kernel?

Reply:
All reasonably recent versions of the plugin (since at least v1.2, probably even older) always set this flag for supported versions of Libfabric, regardless of kernel version. So, the user never needs to set this flag.